### PR TITLE
Chrisdev hover info 1203486711938683

### PR DESCRIPTION
**Pull request by chriscodev**
Hi @triveni-codev , 

i have new updates on this ASANA ticket: 
https://app.asana.com/0/1203486985240966/1203486711938683

fields to check and test transport input hover atsc

PSIP-TERRISTRIAL
  TSID
  EDIT PSI
  EDIT PSIP
    EDIT PROGRAMS
    ELEMENTARY STREAMS
      AC3 AUDIO

ATSC 3
  TSID
  SERVICE GROUPS
    LLS SETTINGS
    ESG SETTING
    AEA SETTINGS
    RECOVERY SETTINGS 
      V1EMBEDDERS

### CODE REVIEW

**Comment by pauljustinhernandez** [comment]
Hi Tian, dropping some issues encountered upon testing;

1. No modal appeared upon clicking the EDIT PSIP TABLE SETTINGS
psip-table

File: src/app/pages/main/network-config/transport/transport.component.ts [hunk 1]
```diff
@@ -2034,6 +2045,12 @@ export class TransportComponent implements OnInit {
       // pass dataservice to atsc transport stream
 
       let tableIdx = this.tableData.findIndex((x) => x.id === this.transportDataObject.id);
+      console.log('tableIdx',tableIdx)
```
Comment (pauljustinhernandez) [inline src/app/pages/main/network-config/transport/transport.component.ts:2048]: Kindly remove console logs, Thanks!

**Comment by pauljustinhernandez** [comment]
LGTM! Tian @chriscodev , just kindly remove console logs if not needed. Thanks!

cc: @triveni-codev

**Comment by chriscodev** [comment]
hi @pauljustinhernandez , i have updates on this PR, you may now verify in your end.

Thanks,
Christian

File: src/app/components/modals/modal-transport/modal-transport-edit-esg-settings/modal-transport-edit-esg-settings.component.html [hunk 2]
```diff
@@ -121,7 +125,7 @@ <h5 class="modal-title" id="myModalLabel">Edit ESG Settings</h5>
                 (keypress)="onInputNumber($event)"
                 (keyup)="validateInputFieldset($event,'hoursBehind')"
                 maxlength="4"
-                title="Max Value 1536"
+                title="Max alue 1536"
```
Comment (pauljustinhernandez) [inline src/app/components/modals/modal-transport/modal-transport-edit-esg-settings/modal-transport-edit-esg-settings.component.html:128]: Kindly double check this Tian, Thanks!

File: src/app/pages/main/network-config/transport/transport.component.ts [hunk 3]
```diff
@@ -1991,17 +1997,22 @@ export class TransportComponent implements OnInit {
...
         }
 
         this.transportDataObject['serviceGroups'][findIndex] = $event.dataRow;
+
         this.modalAC3ElemStreamArr = $event.dataRow;
 
+
         //update ngonchanges to update service groups table
       } else {
         console.log("ELSE",$event.dataRow);
```
Comment (pauljustinhernandez) [inline src/app/pages/main/network-config/transport/transport.component.ts:2018]: remove if not needed

**Review by pauljustinhernandez — COMMENTED** [review]
Hi Tian, couple of minimal concerns here @chriscodev

**Pull request merged**